public/css/home.css

```diff
@@ -0,0 +1,86 @@
+@import url('https://framer-ui.netlify.app/components/framer.css');
+
+:root{
+    --navbar-bg:#2C2929;
+    --img-bg:#2F3136;
+    --img-text:#f1f2f3;
+
+
+}
+
+.navbar-bg{
+    background: var(--navbar-bg);
+}
+
+.nav-zindex{
+    z-index: 9999;
+}
+
+.user-profile{
+    width: 100%;
+    height: 100%;
+    border-radius: 5px;
+    max-height: 11.938rem;
+    object-fit: cover;
+}
+.qg-gap-5{
```

Comment:
This class name is not understandable.

Reply:
Well, this is the **prefix**, for not to collide the actual CSS properties with the component library's  CSS properties, Here basically. **qg** is the initial letter of Quiz and grad